### PR TITLE
Message updates

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: data/discord_messages.yaml [hunk 1]
```diff
@@ -22,43 +22,33 @@
...
-
+    **Trying to install HAOS with an SSD in Rpi4?** Things you need to know:
 
-    1) USB boot is supported only by the 64-bit [5.x dev
-    images](https://github.com/home-assistant/operating-system/releases) as of
-    now. (You installed a 5.x image but it's not working? -> Make sure it's
-    actually the 64-bit version!).
 
-    2)Make sure your Pi is updated to a stable
+    1)Make sure your Pi is updated to a stable
```
Comment (balloob) [inline data/discord_messages.yaml:41]:
```suggestion
    1) Make sure your Pi is updated to a stable
```

Comment (balloob) [reply data/discord_messages.yaml:41]: Shouldn't we just make this "latest bootloader" ?

File: data/discord_messages.yaml [hunk 2]
```diff
@@ -22,43 +22,33 @@
...
     [bootloader](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloader_config.md#usbmassstorageboot)
     release that supports USB boot
 
-    3) If install still fails, then your SSD likely needs [quirks
+    2) If install still fails, then your SSD likely needs [quirks
     enabled](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) to work
     correctly. Find your adapter's ID and add the quirks parameter to your
     `cmdline.txt` file (found in the boot partition) once you've flashed your
-    5.x HassOS image of choice on the SSD.
+    5.x HAOS image of choice on the SSD.
```
Comment (balloob) [inline data/discord_messages.yaml:49]: Should this be 5.x? We're at 9?
```suggestion
    HAOS image for your Raspberry Pi on the SSD.
```

File: data/discord_messages.yaml [hunk 3]
```diff
@@ -466,7 +418,7 @@ split_config:
     Here is now to [split your
     configuration](https://home-assistant.io/docs/configuration/splitting_configuration/)
     into multiple YAML files
-Videos:
+videos:
   description: "Video are often out of date"
```
Comment (balloob) [inline data/discord_messages.yaml:422]:
```suggestion
  description: "Videos are often out of date"
```

File: services/bots/src/discord/commands/message.ts [hunk 4]
```diff
@@ -74,10 +79,16 @@ export class MessageCommand implements DiscordTransformedCommand<MessageDto> {
 
     await this.ensureMessageDataLoaded();
 
+    console.log(this.messageData[messageKey]);
+
```
Comment (balloob) [inline services/bots/src/discord/commands/message.ts:83]:
```suggestion
```

**Review by balloob — APPROVED** [review]

File: data/discord_messages.yaml (hunk 1 above)
Comment (ludeeus) [reply data/discord_messages.yaml]: What does rpi use?

**Pull request merged**